test/views/CreditsSummaryViewSpec.scala

```diff
@@ -40,7 +40,7 @@ class CreditsSummaryViewSpec extends TestSupport with FeatureSwitching with Impl
   lazy val mockAppConfig: FrontendAppConfig = app.injector.instanceOf[FrontendAppConfig]
 
   val testCalendarYear: Int = 2018
-  val expectedDate: String = "2018"
+  val expectedDate: String = "29 Mar 2018"
```

Comment:
still not convinced that we should use String instead of LocalDate, as this is really representing moment in time